src/index.html

```diff
@@ -8,9 +8,36 @@
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./styles/style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      data-qa="big"
+      method="post"
+      class="big_search-bar"
+    >
+      <input
+      class="big_search-bar__input"
+      type="text"
+      placeholder="Try &quot;Los Angeles&quot;"
+      name="big_search"
+      data-qa="keypress"
+      >
```

Comment:
```suggestion
      <input
        class="big_search-bar__input"
        type="text"
        placeholder="Try &quot;Los Angeles&quot;"
        name="big_search"
        data-qa="keypress"
      >
```
Pay attention to the formatting